boards/shields/lcd_par_s035/lcd_par_s035_8080.overlay

```diff
@@ -24,8 +24,8 @@
 	gt911_lcd_par_s035: gt911-lcd_par_s035@5d {
 		compatible = "goodix,gt911";
 		reg = <0x5d>;
-		alt-addr = <0x14>;
```

Comment:
why is this deleted? might want to mention in commit message

Reply:
I've added a blurb in the commit message. For anyone interested, the best I can tell this was working because the FRDM-MCXN947 board (previously the only in tree board using this display) was configuring the INT pin for the display in the MIPI-DBI pincontrol node. The pin control setting was such that when the RESET pin for the display (and touch screen) controller is toggled low, the touch screen controller will select the 0x14 alternate I2C address. This workaround wasn't working on the RW612 board, so I opted to properly set initialization order, thereby making sure the GT911 driver is able to pull the reset pin low and set the INT pin properly as needed to get the GT911 IC to select the standard 0x5D I2C address.